odl/test/solvers/functional/default_functionals_test.py

```diff
@@ -544,7 +544,7 @@ def test_weighted_proximal_L2_norms_squared(space):
     assert all_almost_equal(p1, p2)
 
     # Check if the subdifferential inequalities are satisfied.
-    # p = prox_{σf}(x) ⇔ (x - p)/σ = ∇f(p)
+    # p = prox_{sigma * f}(x) iff (x - p)/sigma = grad f(p)
```

Comment:
Agreed that we should use ASCII when possible, to avoid hassle. But when we e.g. cite authors with non-ASCII characters in their names, I don't think the best solution would be to misspell their names. 
Instead, according to [PEP 263](https://www.python.org/dev/peps/pep-0263/) one can put a comment `# coding: utf-8` (I'd prefer this style) into the *first or second line* of the file so that at least parsing the source works for Python 2.7 and 3. 
The characters may still come out wrong in strings, but that can be fixed with `from __future__ import unicode_literals`. This option is a bit dangerous, though, especially when passing strings to other libraries.

Reply:
I strongly oppose using unicode in code as long as we support python 2. We're bound to get problems at some point from doing this, not to mention the irritating bugs one can get from unicode characters that look EXACTLY the same

Reply:
Docstrings are not code, at least not Python code. And PEP 263 is implemented since Python 2.3.

Reply:
Still sceptical. Editor support for Unicode is sketchy at best.

Either way, if we're going to allow Unicode that should have its own issue and discussion and not be added as an afterthought in some PR.

If it can safely be done, I'm all for it. But we need to check all major editors etc. We also need rules on where we allow it (i.e. only in docstrings?)

Reply:
Sure, this is not the right place for discussion. 

But anyway, what I wrote is not about editors at all. It is about the Python interpreter that has to process a code file, and that can be reliably steered with the comment according to the PEP.

See #1207